react/Viewer/PdfJsViewer.spec.jsx

```diff
@@ -112,7 +112,7 @@ describe('PDFViewer', () => {
     })
 
     it('should render all the pages', () => {
-      const pages = component.find('Page')
+      const pages = component.find('.page-react-pdf')
```

Comment:
Added class to make test work, I'm at a loss here why the test doesn't work with Component selector but works with class selector

Reply:
because you upgraded the whole yarn.lock => 

```
react-pdf@^4.0.5:
  version "4.2.0"
  resolved "https://registry.yarnpkg.com/react-pdf/-/react-pdf-4.2.0.tgz#b83a01eb070912522075b7a51aee7d63b2c912ed"
  integrity sha512-Ao44mZszfPwtCUsrXHtXnhM+czTvPxfG5wqssbWgj2onL70TKSOKGzQfCH4csCnNDOKji/Pc/s0Og70/VOE+Rg==
  dependencies:
    "@babel/runtime" "^7.0.0"
    make-cancellable-promise "^1.0.0"
    make-event-props "^1.1.0"
    merge-class-names "^1.1.1"
    pdfjs-dist "2.1.266"
    prop-types "^15.6.2"
 ```
 
 i advice you to upgrade only the needed package 🙏

Reply:
I had no choice the frozen-lockfile was busted. I did not remake the lock file though, only `yarn install`. What do you think then

Reply:
then use only `react-pdf@4.0.5`

Reply:
and try to use every package without the ^ (in order not to upgrade version) / keep the same version